src/arch-syscall-validate

```diff
@@ -911,6 +942,7 @@ if [[ $opt_arches == "" ]]; then
 		x86 x86_64 x32 \
 		arm aarch64 \
 		loongarch64 \
+		arm aarch64 m68k \
```

Comment:
super nitpick - this line should just be `m68k`, correct, and not `arm aarch64 m68k`?

Reply:
Thanks, just fixed this. Obvious copy-and-paste error.